client/cmd_test.go

```diff
@@ -38,10 +37,10 @@ func TestValidateCmd(t *testing.T) {
 		args    []string
 		wantErr bool
 	}{
-		{"misspelled command", []string{"comission"}, true},
+		{"misspelled command", []string{"comission"}, true}, // nolint: misspell
 		{"no command provided", []string{}, false},
-		{"help flag", []string{"comission", "--help"}, false},
-		{"shorthand help flag", []string{"comission", "-h"}, false},
+		{"help flag", []string{"commission", "--help"}, false},       // nolint: misspell
```

Comment:
`commission` needs to go back to being misspelled lol, the test is checking if when a command is misspelled and the help flag is used, that an error is not thrown, but if the command is spelled correctly it definitely won't throw an error.

spelled correctly -> no error
misspell -> error
misspell + help_flag -> no error